tests/citeproc.rs

```diff
@@ -597,6 +620,153 @@ where
     }
 }
 
+/// Functions to format bibliography rendered by hayagriva using citeproc's HTML
+/// output format, in order to be able to compare the generated HTML.
+mod citeproc_bib {
+    use core::fmt;
+
+    use citationberg::{Display, FontStyle, FontVariant, FontWeight, VerticalAlign};
+    use hayagriva::{BufWriteFormat, Elem, ElemChild, Formatting};
+
+    pub(super) fn render(
+        bib: &hayagriva::RenderedBibliography,
+        output: &mut String,
+    ) -> Result<(), fmt::Error> {
+        output.push_str(r#"<div class="csl-bib-body">"#);
+        for item in &bib.items {
+            // TODO: Verify whether this automatically implies left-margin
+            // followed by right-inline (cf. test bugreports_AsmJournals.txt)
```

Comment:
The `left-margin`, `right-inline` in `bugreports_AsmJournals.txt` is caused by the [`second-field-align="true"`](https://docs.citationstyles.org/en/stable/specification.html#whitespace) attribute on the `bibliography` element

Reply:
Implemented, thank you.